lib/CXGN/BrAPI/v2/Locations.pm

```diff
@@ -270,7 +277,9 @@ sub store {
 		my $store = $location->store_location();
 
 		if ($store->{'error'}) {
```

Comment:
Is it possible to also show this original error in the warning?

Reply:
ya, that'd be a good improvement. Pushed changes.